src/engine/sync/enginesync.cpp

```diff
@@ -533,7 +531,7 @@ void EngineSync::onCallbackEnd(int sampleRate, int bufferSize) {
     m_pInternalClock->onCallbackEnd(sampleRate, bufferSize);
 }
 
-EngineChannel* EngineSync::getLeader() const {
+EngineChannel* EngineSync::getLeaderChannel() const {
```

Comment:
Looks unrelated, but but probably makes sense.

Reply:
yeah it helped clarify -- "getLeader" would seemingly return a Syncable*